### PR TITLE
feat: split up multi hop ui components into files

### PR DESCRIPTION
**Pull request by woodenfurniture**
## Description

Splits up existing multi hop ui components into the standard directory structure.

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

NA

## Risk

Extremely low risk as this is behind a feature flag.

## Testing

Not required as this is behind a feature flag.

### CODE REVIEW

**Comment by woodenfurniture** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #5567** <a href="https://app.graphite.dev/github/pr/shapeshift/web/5567" target="_blank">Graphite</a> 
    * **PR #5575** <a href="https://app.graphite.dev/github/pr/shapeshift/web/5575" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/5575?utm_source=stack-comment).

File: src/components/MultiHopTrade/components/MultiHopTradeConfirm/components/FirstHop.tsx [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+  const shouldRenderDonation = true // TODO:
+  const { onRejectApproval, onSignTrade, onRejectTrade } = useTradeExecutor()
+
+  const tradeExecutionStatus = useAppSelector(selectTradeExecutionStatus)
+
+  const activeStep = useMemo(() => {
+    switch (tradeExecutionStatus) {
+      case MultiHopExecutionStatus.Unknown:
+        return -Infinity
```
Comment (gomesalexandre) [inline src/components/MultiHopTrade/components/MultiHopTradeConfirm/components/FirstHop.tsx:62]: 🤯

**Review by gomesalexandre — APPROVED** [review]
Stamping as this is mostly moved bits and is under flag

File: src/components/MultiHopTrade/components/MultiHopTradeConfirm/hooks/useTradeExecutor.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { useCallback } from 'react'
+import { swappers } from 'state/slices/swappersSlice/swappersSlice'
+import { useAppDispatch } from 'state/store'
+
+export const useTradeExecutor = () => {
```
Comment (gomesalexandre) [inline src/components/MultiHopTrade/components/MultiHopTradeConfirm/hooks/useTradeExecutor.tsx:5]:
```suggestion
export const useTradeExecutooor = () => {
```

Comment (woodenfurniture) [reply src/components/MultiHopTrade/components/MultiHopTradeConfirm/hooks/useTradeExecutor.tsx]: 0d95466eb7 🫡

**Pull request merged**